source/text/glyphvertices.ts

```diff
@@ -140,10 +157,10 @@ export class GlyphVertices {
     }
 
     /**
-     * All GlyphVertex up vectors gathered in one buffer. Call update() to update it.
+     * All GlyphVertex up/bitangent vectors gathered in one buffer. Call update() to update it.
```

Comment:
doc: there is no `update()` anymore. Same for origins, tangents, bitangents and texCoords.